src/leiningen/monolith.clj

```diff
@@ -181,13 +183,61 @@
   (checkouts/unlink project))
 
 
+;; ## Fingerprinting
+
+;; Leiningen's help framework doesn't officially support "subsubtasks" so
+;; we'll try our best to make it work by making them look like subtasks.
```

Comment:
Seems like you've promoted them to full subtasks now, so this comment may be out of date.

Reply:
Oops, yeah